File: fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/reader/ReaderSearchSitesResponse.java
package org.wordpress.android.fluxc.network.rest.wpcom.reader;

import android.support.annotation.NonNull;

import com.google.gson.annotations.JsonAdapter;

import org.wordpress.android.fluxc.model.ReaderSiteModel;

import java.util.List;

@JsonAdapter(ReaderSearchSitesDeserializer.class)
public class ReaderSearchSitesResponse {
    public @NonNull List<ReaderSiteModel> sites;
}

File: fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/reader/ReaderSearchSitesResponse.kt
package org.wordpress.android.fluxc.network.rest.wpcom.reader

import com.google.gson.annotations.JsonAdapter

import org.wordpress.android.fluxc.model.ReaderSiteModel

@JsonAdapter(ReaderSearchSitesDeserializer::class)
class ReaderSearchSitesResponse (
    val offset: Int,
    val sites: List<ReaderSiteModel>
)
